### ISSUE TITLE
Unexpected output with --show_testcase and --test module/method_name

### ISSUE THREAD

**Issue by vlevigneron** [zonemaster/zonemaster-engine#970]
zonemaster-cli --level INFO --show_testcase  --locale en_US.UTF-8 --test dnssec/dnssec15 nic.fr

> Seconds Level     Testcase       Message
======= ========= ============== =======
   0.00 INFO      UNSPECIFIED    Using version v4.2.3 of the Zonemaster engine.
   2.16 INFO      UNSPECIFIED    No CDS or CDNSKEY RRsets are found on any name server.

Testcase value is UNSPECIFIED

When no method_name is specified, here is the result
zonemaster-cli --level INFO --show_testcase  --locale en_US.UTF-8 --test dnssec nic.fr

>Seconds Level     Testcase       Message
======= ========= ============== =======
   0.00 INFO      UNSPECIFIED    Using version v4.2.3 of the Zonemaster engine.
   1.49 INFO      DNSSEC01       d.nic.fr/194.0.9.1 returned a DS record created by algorithm 2 (SHA-256), which is OK. The DS record is for the DNSKEY record with keytag 21858 in zone nic.fr.
   1.50 INFO      DNSSEC01       d.nic.fr/2001:678:c::1 returned a DS record created by algorithm 2 (SHA-256), which is OK. The DS record is for the DNSKEY record with keytag 21858 in zone nic.fr.
   1.52 INFO      DNSSEC01       e.ext.nic.fr/193.176.144.22 returned a DS record created by algorithm 2 (SHA-256), which is OK. The DS record is for the DNSKEY record with keytag 21858 in zone nic.fr.
[...]

Testcase value is correct

**Comment by matsduf** [comment]
Yes, it is a known issue, but I feel it has low priority, but if it is very simple to fix then it is good to have it fixed.
